release.yaml

```diff
@@ -15,9 +15,11 @@ rancher-alerting-drivers:
 rancher-backup:
 - 1.2.100+up1.2.1
 - 2.1.1+up2.1.1-rc1
+- 2.1.1+up2.1.1-rc2
 rancher-backup-crd:
 - 1.2.100+up1.2.1
 - 2.1.1+up2.1.1-rc1
+- 2.1.1+up2.1.1-rc2
```

Comment:
Since `2.1.1+up2.1.1-rc1` is dropped in this PR, I guess we should remove it from the list?

Reply:
No, we can't. Validation fails. All old RCs must appear here.

Reply:
OK. 
LGTM if that is how chart-builder works today.